src/test/java/org/jboss/tools/intellij/openshift/utils/MessagesHelper.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
```

Comment:
Why is this class duplicated ? That breaks the DRY rule

Reply:
> Why is this class duplicated ? That breaks the DRY rule

as explained, using this class from intellij-common breaks the tests compilation and the build.
moving to fabric8 implies to refactor code ( 4.12.0 -> 5.0.0 ) and could be achieve with #234 for 0.5.0 .

Reply:
Then we should wait until this is done otherwise we will have code duplicated and forget to remove it